stable/kibana/values.yaml

```diff
@@ -1,70 +1,49 @@
+replicaCount: 1
+
 image:
   repository: "docker.elastic.co/kibana/kibana-oss"
-  tag: "6.0.0"
+  tag: "6.2.4"
   pullPolicy: "IfNotPresent"
+#  pullSecrets:
 
-commandline:
-  args:
-
-env:
-  # All Kibana configuration options are adjustable via env vars.
-  # To adjust a config option to an env var uppercase + replace `.` with `_`
-  # Ref: https://www.elastic.co/guide/en/kibana/current/settings.html
-  #
-  # ELASTICSEARCH_URL: http://elasticsearch-client:9200
-  # SERVER_PORT: 5601
-  # LOGGING_VERBOSE: "true"
-  # SERVER_DEFAULTROUTE: "/app/kibana"
+ports:
+  kibana:
+    containerPort: 5601
 
 service:
   type: ClusterIP
-  externalPort: 443
-  internalPort: 5601
-  ## External IP addresses of service
-  ## Default: nil
-  ##
-  # externalIPs:
-  # - 192.168.0.1
-  #
-  ## LoadBalancer IP if service.type is LoadBalancer
-  ## Default: nil
-  ##
-  # loadBalancerIP: 10.2.2.2
-  annotations:
-    # Annotation example: setup ssl with aws cert when service.type is LoadBalancer
-    # service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-east-1:EXAMPLE_CERT
+  annotations: {}
+  ports:
+    kibana:
+      port: 5601
+      targetPort: 5601
 
 ingress:
   enabled: false
-  # hosts:
-    # - chart-example.local
-  # annotations:
-  #   kubernetes.io/ingress.class: nginx
-  #   kubernetes.io/tls-acme: "true"
-  # tls:
-    # - secretName: chart-example-tls
-    #   hosts:
-    #     - chart-example.local
```

Comment:
I realize it's cleaner to remove such commented text, but it was very helpful for me when I was getting started.